src/components/molecules/NavBarScheduleWeather/WeatherBackground/BlackRockWeatherBackground.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { PropsWithChildren, useState } from "react";
+import useInterval from "react-use/lib/useInterval";
+import classNames from "classnames";
+import dayjs from "dayjs";
+import timezone from "dayjs/plugin/timezone";
+import utc from "dayjs/plugin/utc";
+
+import { ContainerClassName } from "types/utility";
+
+import "./WeatherBackground.scss";
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+
+const PART_OF_DAY = ["night", "morning", "day", "evening"];
+const PART_OF_DAY_LENGTH = 24 / PART_OF_DAY.length;
+const IMAGES_PER_PART_OF_DAY = 3;
+
+const IMAGE_UPDATE_INTERVAL = 60 * 1000;
+
+const getCurrentImageName = () => {
+  //timezone of Black Rock Desert
+  const time = dayjs().tz("America/Los_Angeles");
+  const hour = time.hour();
+  console.log("hour", hour);
```

Comment:
If you mean `useEffect` cleanup function - then yes.
There is this line inside `useInterval`
```typescript
return () => clearInterval(id);
```

Also removed `console.log`s in https://github.com/sparkletown/sparkle/pull/2092/commits/dd9a0cecab17b3df400224c5cc96bfe1c5737f46

Reply:
Should this response be enough for https://github.com/sparkletown/sparkle/pull/2092#discussion_r698291808 ?

So far so good, let's just keep an eye how the weather widgets behaves in case we've missed some corner case.